.ci/jobs/defaults.yml

```diff
@@ -11,21 +11,19 @@
     view: APM-CI
     project-type: multibranch
     logrotate:
-      daysToKeep: 30
       numToKeep: 100
-    number-to-keep: '5'
-    days-to-keep: '1'
     concurrent: true
     node: linux
     script-path: Jenkinsfile
     scm:
     - github:
-        branch-discovery: all
+        branch-discovery: no-pr
         discover-pr-forks-strategy: merge-current
         discover-pr-forks-trust: permission
         discover-pr-origin: merge-current
         discover-tags: true
         notification-context: 'apm-ci'
+        head-filter-regex: '^(?!module/).*$'
```

Comment:
This is the regex to ensure branches/tags which are not matching `^module/` should not be ignored (sorry about my double negation!)  :)